modules/apps/portal-security-sso/portal-security-sso-openid-connect-impl/src/main/java/com/liferay/portal/security/sso/openid/connect/internal/OpenIdConnectAuthenticationHandlerImpl.java

```diff
@@ -137,8 +138,37 @@ public void processAuthenticationResponse(
 			_getLoginRedirectURI(httpServletRequest),
 			oAuthClientEntry.getTokenRequestParametersJSON());
 
-		String userInfoJSON = _requestUserInfoJSON(
-			oidcTokens.getAccessToken(), oidcProviderMetadata);
+		URI userInfoURI = oidcProviderMetadata.getUserInfoEndpointURI();
+
+		String userInfoJSON = null;
+
+		if (userInfoURI == null) {
```

Comment:
Maybe it actually would be better to use `Validator#isNotBlank` ?

Reply:
the isBlank that you mentioned accepts only String


how do you like this? if (Validator.isNull(userInfoURI)) {